.rubocop_todo.yml

```diff
@@ -0,0 +1,2 @@
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
This will be needed for the ruby 3 release due this december.

While we won't be consuming that any time soon, the changes for this are relatively straight-forward and between auto_correct and the unit tests we need to take the time to fix those issues now.

Reply:
sure. Let me try it and see the errrors its populating.